packages/css-library/src/stylesheets/_core.scss

```diff
@@ -0,0 +1,48 @@
+@use "../../dist/tokens/scss/variables" as *;
+
+@use 'uswds-core' with (
+  $theme-show-notifications: false,
+  // TODO: verify USWDS font path
+  $theme-font-path: '../node_modules/@uswds/uswds/dist/fonts',
+);
+
+$font-path: "./fonts";  // TODO: Verify asset path
+$image-path: "./img"; // TODO: Verify asset path
```

Comment:
These asset paths point to where the assets will be in `dist/` but when we start connecting stylesheets, this will need to be verified that it's correct.